### ISSUE TITLE
[POC] How to set up focus ring in Figma

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#464]
Find a good setup for the focus ring which we can used in all focus states in the component variants within Figma.

```[tasklist]
### What do we want to achieve?
- [x] Focus ring should be implemented in the component states (it is not a new component)
- [x] Focus state should be usable in prototypes (it should not jump a few px when it goes from 'idle' to 'focus')
- [x] we prefer to have the focus ring on the outside of the element instead of the inside (microsoft fluent)
- [x] make sure contrast ratio is acceptable
- [x] Only one color is applied by default but can be overruled when there is a good reason
- [x] focus state can be applied on all states and you can turn it on and off
```
